Source/common/SNTConfigurator.h

```diff
@@ -72,6 +72,54 @@
 ///
 @property(readonly, nonatomic) NSRegularExpression *fileChangesRegex;
 
+///
+///  A list of ignore prefixes which are checked in-kernel.
+///  This is more performant than FileChangesRegex when ignoring whole directory trees.
+///
+///  For example adding a prefix of "/private/tmp/" will turn off file change log generation
+///  in-kernel for that entire tree. Since they are ignored by the kernel, they never reach santad
+///  and are not seen by the fileChangesRegex. Note the trailing "/", without it any file or
+///  directory starting with "/private/tmp" would be ignored.
+///
+///  By default "/." and "/dev/" are added.
+///
+///  Memory in the kernel is precious. A total of 256 bytes are allowed for this configuration.
+///  An ASCII character uses 1 byte. An UTF-8 encoded Unicode character uses 1-4 bytes.
+///  Prefixes are added to the running config in-order one by one. The prefix will be ignored if
+///  (the running config's current size) + (the prefix's size) totals up to more than 256 bytes.
+///  The running config is stored in a prefix tree. Each node in the tree represents 1 byte.
+///  Prefixes that share prefixes are effectively de-duped; their shared byte sized components only
+///  take up 1 byte. For example these 3 prefixes all have a common prefix of "/private/".
+///  They will only take up 21 bytes instead of 39.
```

Comment:
Should this be 21 nodes instead of 21 bytes?  Otherwise it seems a bit misleading since each node is actually  1 byte for bool + 256 * 8 bytes for pointer array = 2 kilobytes